LICENSE

```diff
@@ -0,0 +1,30 @@
+Copyright OpenSearch Contributors.
+
+Copyright (c) Django Software Foundation and individual contributors.
```

Comment:
I think we can remove this
and credit `vitepress` (Evan You)
https://vitepress.dev/

Reply:
Good catch, resolved.